src/NCalc.Core/ExpressionEvaluator.cs

```diff
@@ -41,7 +41,8 @@ public int CompareUsingMostPreciseType(object? a, object? b)
     
     protected MathHelperOptions MathHelperOptions => new(Context.CultureInfo,
```

Comment:
Perfect, but you need to do the same thing at `AsyncExpressionEvaluator`

Reply:
Nice catch, I'll fix it tomorrow evening (I'm on the phone now)

Reply:
No need, for the change, this is on the base class for both, sorry